sonar-plugin-api/src/main/java/org/sonar/api/server/ws/internal/ValidatingRequest.java

```diff
@@ -49,6 +51,15 @@ public void setAction(WebService.Action action) {
   }
 
   @Override
+  public LocalConnector getLocalConnector() {
+    return localConnector;
+  }
+
+  public void setLocalConnector(LocalConnector lc) {
```

Comment:
Return value can't be null:
* ValidatingRequest checks that it is not null
* SimpleGetRequest throws UnsupportedOperationException